gloo/docs/gloo_routing/virtual_services/authentication/plugin_auth/_index.md

```diff
@@ -274,7 +281,7 @@ glooctl add route --path-prefix /echo --dest-name default-http-echo-5678
 To verify that the Virtual Service works, let's get the URL of the Gloo Gateway and send a request to `/echo`:
 
 ```bash
-export GATEWAY_URL=$(glooctl proxy url)
+export GATEWAY_URL=$(glooctl proxy url --name gateway-proxy-v2)
```

Comment:
will this be necessary once https://github.com/solo-io/gloo/issues/1123 is fixed? if not, probably better to omit so these docs don't get outdated once a v3 is released

Reply:
It will not be necessary, just redundant. Not sure how soon that will be fixed, so I prefer the redundancy than the error users would get until it is fixed.